tests/kola/test-coreos.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
```

Comment:
Any reason to keep this under kola directory? Do you anticipate adding any other tests?

Reply:
I don't. I was keeping form with kola external test convention, but this is not really an external test in cosa terms, so we can totally loose this and just put it on tests folder.